### ISSUE TITLE
[Bug]: N'affiche pas la page d'accueil

### ISSUE THREAD

**Issue by Iandesuu** [PapillonApp/papillon-v5#92]
### Description du bug

Je ne peux pas voir la page d'accueil alors que les autres onglets comme l'emploi du temps et la note marche parfaitement. Quand je reviens dans l'appli quelques heures apres la dernier ouverture, ça affiche pendant 1 seconde ce qu'il devrait afficher.

### Etapes à reproduire

Il n'y a pas d'étapes a réproduire. ça me fais à chaque fois.

### Comportement attendu

Ne plus afficher "chargement de vos données" et afficher l'accueil correctement

### Appareil

Google Pixel 3

### Système d`exploitation

Android

### Version

12

### Environnement

Application native Android

### Service scolaire

🦋 Pronote

### Captures d'écran / vidéo

https://user-images.githubusercontent.com/113456382/230183790-0e1900fa-c886-400c-81ad-f0c96b89559b.mp4

### Contexte supplémentaire

Même probleme avec le telephone de ma soeur. (Google Pixel 3) - aussi pronote mais different établissement

pas de message d'erreur.

**Comment by TheTiger72** [comment]
Tu as quelle version de Papillon ?

**Comment by Iandesuu** [comment]
J'ai actuellement le "version 5.3.6-1-g0396903" qui est je pense un beta mais j'avais ce probleme meme avant que je m'insctive pour le testeur beta.

**Comment by sivelswhy** [comment]
Essaye d'envoyer tes logs (trouvable dans les paramètres), peut-être que ça pourra les aider

**Comment by Iandesuu** [comment]
voici le log:

Le **5 avril 2023 à 23:34**
Contient **100** logs

*Application* :
> **Version** : 5.3.6-1-g0396903
> **Canal** : Bêta
> **Type de plateforme** : android
> **Version API** : 3.4.4
> **Taille du cache** : 48.49 kb

*Appareil* :
> **ID** : 93b37293eb2b6c20
> **Modèle** : Pixel 3
> **Version OS** : 12
> **Marque** : Google

*Compte* :
> **Nom** : ####
> **Établissement** : ####
> **URL** : https://0770934x.index-education.net/pronote/eleve.html
> **CAS** : ile_de_france


**Voir les logs** : *https://logs.getpapillon.xyz/zaqojebahi*
> NB. Les logs sont supprimés après 48h de nos serveurs.

**Comment by tom-theret** [comment]
Bonjour, le bug est t-il toujours présent ?

**Comment by Iandesuu** [comment]
Oui, je viens de verifier si il étais a jour puis testé mais le bug est toujours present.

**Comment by tom-theret** [comment]
Tu peux faire un partage de tes logs s’il te plaît (les autres sont expirer.) ?

**Comment by Iandesuu** [comment]
ah et comme de par hasard le probleme est résolu!
a voir si il ne revienne pas.

**Comment by Iandesuu** [comment]
C'étais bien temporaire, le problème est revenu!

je met une video de mon écran et le log:

https://user-images.githubusercontent.com/113456382/231145843-57f012fd-05ca-46c7-8266-ceca70ae26a4.mp4

https://user-images.githubusercontent.com/113456382/231153114-a9fda9fa-fd05-4f00-b1a6-442adb95f315.mp4


Le **11 avril 2023 à 13:46**
Contient **100** logs

*Application* :
> **Version** : 5.4.0-4-g11dcfd5
> **Canal** : Stable
> **Type de plateforme** : android
> **Version API** : 3.4.4
> **Taille du cache** : 99.45 kb

*Appareil* :
> **ID** : 93b37293eb2b6c20
> **Modèle** : Pixel 3
> **Version OS** : 12
> **Marque** : Google

*Compte* :
> **Nom** : #############
> **Établissement** : ############
> **URL** : https://0770934x.index-education.net/pronote/eleve.html
> **CAS** : ile_de_france


**Voir les logs** : *https://logs.getpapillon.xyz/ibamegarel*
> NB. Les logs sont supprimés après 48h de nos serveurs.
